### ISSUE TITLE
Supertable migration issue after updating to Craft 3

### ISSUE THREAD

**Issue by jwrudzin** [engram-design/SuperTable#285]
### Description

We updated our site from Craft 2 to Craft 3, and are getting the following error when we run the Super Table migration:

Database Exception: SQLSTATE[42000]: Syntax error or access violation: 1072 Key column 'locale' doesn't exist in table
The SQL being executed was: ALTER TABLE craft_stc_32_sessions DROP COLUMN locale

### Steps to reproduce

1. Update Craft site from 2 -> 3
2. Install Supertable from composer command line
3. Click "finish up" in Craft CP

### Additional info

- Plugin version: 2.2.1
- Craft version: 3.2.8

**Comment by johnnynotsolucky** [comment]
I had a similar issue with a migration from Craft 2 to 3.

In my case its with `craft_stc_3_boxcontentoptions`, `craft_stc_giftcardamounts`, `craft_stc_shippingtimeline`, and a few other tables.

Running `SHOW CREATE TABLE craft_stc_3_boxcontentoptions`, the constraint in this DB is `craft_supertableconten_3_boxcontentoption_elementI_locale_unq_id`, and searching for the constraints yields:

```sql
SELECT 
	TABLE_NAME, COLUMN_NAME, CONSTRAINT_NAME, REFERENCED_TABLE_NAME, REFERENCED_COLUMN_NAME 
FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE 
WHERE constraint_name = 'craft_supertableconten_3_boxcontentoption_elementI_locale_unq_id'
```
```
+---------------------------------------------+-------------+------------------------------------------------------------------+-----------------------+------------------------+
| TABLE_NAME                                  | COLUMN_NAME | CONSTRAINT_NAME                                                  | REFERENCED_TABLE_NAME | REFERENCED_COLUMN_NAME |
+---------------------------------------------+-------------+------------------------------------------------------------------+-----------------------+------------------------+
| craft_supertablecontent_3_boxcontentoptions | id          | PRIMARY                                                          | NULL                  | NULL                   |
| craft_supertablecontent_3_boxcontentoptions | elementId   | craft_supertableconten_3_boxcontentoption_elementI_locale_unq_id | NULL                  | NULL                   |
| craft_supertablecontent_3_boxcontentoptions | locale      | craft_supertableconten_3_boxcontentoption_elementI_locale_unq_id | NULL                  | NULL                   |
| craft_supertablecontent_3_boxcontentoptions | elementId   | craft_supertablecontent_3_boxcontentoptions_elementId_fk         | craft_elements        | id                     |
| craft_supertablecontent_3_boxcontentoptions | locale      | craft_supertablecontent_3_boxcontentoptions_locale_fk            | craft_locales         | locale                 |
+---------------------------------------------+-------------+------------------------------------------------------------------+-----------------------+------------------------+
```

The tables referencing those keys didn't exist anymore. I couldn't find where they came from, perhaps a super early version of Supertable?

I worked around it by temporarily disabling foreign key checks and then dropping the indices and keys in the problem tables.

```sql
set foreign_key_checks = 0;
show create table craft_stc_3_boxcontentoptions;
alter table craft_stc_3_boxcontentoptions drop index craft_supertableconten_3_boxcontentoption_elementI_locale_unq_id;

alter table craft_stc_3_boxcontentoptions drop key craft_supertablecontent_3_boxcontentoptions_locale_fk;

-- Do for every table
```

**Comment by engram-design** [comment]
I think this should be fixed in [2.3.0](https://github.com/verbb/super-table/releases/tag/2.3.0).
